--- gpsbabel-gpsbabel_1_9_0/gui/map.cc.orig	2024-06-25 22:10:16.661252615 +0200
+++ gpsbabel-gpsbabel_1_9_0/gui/map.cc	2024-06-25 22:12:00.294024521 +0200
@@ -92,7 +92,7 @@ Map::Map(QWidget* parent,
   // 1. In the file system in the same directory as the executable.
   // 2. In the Qt resource system.  This is useful if the resource was compiled
   //    into the executable.
-  QString baseFile =  QApplication::applicationDirPath() + "/gmapbase.html";
+  QString baseFile =  "/usr/share/gpsbabel/gmapbase.html";
   QString fileName;
   QUrl baseUrl;
   if (QFile(baseFile).exists()) {
