--- gpsbabel-1.5.1/gui/map.cc.orig	2013-12-31 02:01:14.000000000 +0100
+++ gpsbabel-1.5.1/gui/map.cc	2014-11-23 10:52:00.923720369 +0100
@@ -60,7 +60,7 @@
   connect(this,SIGNAL(loadFinished(bool)),
 	  this,SLOT(loadFinishedX(bool)));
   this->logTimeX("Start map constuctor");
-  QString baseFile =  QApplication::applicationDirPath() + "/gmapbase.html";
+  QString baseFile = "/usr/share/gpsbabel/gmapbase.html";
   if (!QFile(baseFile).exists()) {
     QMessageBox::critical(0, appName,
 			  tr("Missing \"gmapbase.html\" file.  Check installation"));
