--- gpsbabel-1.4.3/Makefile.in.orig	2012-02-19 14:15:39.111366297 +0100
+++ gpsbabel-1.4.3/Makefile.in	2012-02-19 14:28:38.421392392 +0100
@@ -98,7 +98,7 @@
           uuid.o formspec.o xmltag.o cet.o cet_util.o fatal.o rgbcolors.o \
 	  inifile.o garmin_fs.o gbsleep.o units.o @GBSER@ gbser.o \
 	  gbfile.o parse.o avltree.o session.o \
-	$(PALM_DB) $(GARMIN) $(JEEPS) $(SHAPE) @ZLIB@ $(FMTS) $(FILTERS)
+	$(PALM_DB) $(GARMIN) $(JEEPS) $(FMTS) $(FILTERS) $(SHAPE) @ZLIB@
 OBJS = main.o globals.o $(LIBOBJS) @FILEINFO@
 
 .c.o:
