--- gpsbabel-1.5.4/Makefile.in.orig	2018-11-01 11:49:25.061806677 +0100
+++ gpsbabel-1.5.4/Makefile.in	2018-11-01 11:51:00.111159677 +0100
@@ -119,7 +119,7 @@
 	  src/core/xmlstreamwriter.o \
 	  src/core/usasciicodec.o\
 	  src/core/ziparchive.o \
-	  $(GARMIN) $(JEEPS) $(SHAPE) @ZLIB@ $(FMTS) $(FILTERS)
+	  $(GARMIN) $(JEEPS) $(FMTS) $(FILTERS) $(SHAPE) @ZLIB@
 OBJS = main.o globals.o $(LIBOBJS) @FILEINFO@
 
 DEPFILES = $(OBJS:.o=.d)
