diff -ruNp gpsbabel-1.6.0.orig/Makefile.in gpsbabel-1.6.0/Makefile.in
--- gpsbabel-1.6.0.orig/Makefile.in	2019-05-02 10:26:47.000000000 +0200
+++ gpsbabel-1.6.0/Makefile.in	2019-08-06 03:12:10.122585490 +0200
@@ -105,7 +105,7 @@ JEEPS=jeeps/gpsapp.o jeeps/gpscom.o \
 # Extra modules in Jeeps that we don't use
 # 	jeeps/gpsfmt.o jeeps/gpsinput.o jeeps/gpsproj.o
 
-SHAPE=shapelib/shpopen.o shapelib/dbfopen.o shapelib/safileio.o
+SHAPE=-lshp
 
 ZLIB=zlib/adler32.o zlib/compress.o zlib/crc32.o zlib/deflate.o zlib/inffast.o \
 	zlib/inflate.o zlib/infback.o zlib/inftrees.o zlib/trees.o \
@@ -909,11 +909,7 @@ sbp.o: sbp.cc defs.h config.h zlib/zlib.
 session.o: session.cc defs.h config.h zlib/zlib.h zlib/zconf.h cet.h \
   inifile.h gbfile.h session.h src/core/datetime.h src/core/optional.h
 shape.o: shape.cc defs.h config.h zlib/zlib.h zlib/zconf.h cet.h \
-  inifile.h gbfile.h session.h src/core/datetime.h src/core/optional.h \
-  shapelib/shapefil.h
-shapelib/dbfopen.o: shapelib/dbfopen.c shapelib/shapefil.h
-shapelib/safileio.o: shapelib/safileio.c shapelib/shapefil.h
-shapelib/shpopen.o: shapelib/shpopen.c shapelib/shapefil.h
+  inifile.h gbfile.h session.h src/core/datetime.h src/core/optional.h
 skyforce.o: skyforce.cc defs.h config.h zlib/zlib.h zlib/zconf.h cet.h \
   inifile.h gbfile.h session.h src/core/datetime.h src/core/optional.h
 skytraq.o: skytraq.cc defs.h config.h zlib/zlib.h zlib/zconf.h cet.h \
diff -ruNp gpsbabel-1.6.0.orig/shape.cc gpsbabel-1.6.0/shape.cc
--- gpsbabel-1.6.0.orig/shape.cc	2019-05-02 10:26:47.000000000 +0200
+++ gpsbabel-1.6.0/shape.cc	2019-08-06 03:12:55.459376594 +0200
@@ -25,7 +25,7 @@
 #include <QtCore/QVector>
 
 #include "defs.h"
-#include "shapelib/shapefil.h"
+#include <shapefil.h>
 #include <cstdlib>
 
 #if SHAPELIB_ENABLED
